README.md

```diff
@@ -0,0 +1,55 @@
+# ![Reactotron Logo](./docs/plugin-docs/images/readme/Reactotron-128.png)
+
+[Join our Community Slack](http://community.infinite.red/)
+
+# Introduction
+
+Reactotron is a powerful debugger for React and React Native applications. It provides an easy-to-use interface for developers to monitor their application's **state, network requests, and performance metrics** and can be used for any size of project, from small personal apps to large-scale enterprise applications. The OG debugger [Infinite Red](https://infinite.red) team uses on a day-to-day basis to build client apps. Additionally, Reactotron is completely open source and free to use, making it an invaluable tool for developers at all levels of experience.
+
+We recommend to watch [Darin Wilson's](https://github.com/darinwilson) talk at [Chain React](https://chainreactconf.com/): [Chain React 2018: Debugging and Beyond with Reactotron](https://www.youtube.com/watch?v=UiPo9A9k7xc)!
+
+## Installation
+
+On the [Releases](https://github.com/infinitered/reactotron/releases/tag/v2.17.1) page, you can find the latest version of:
+
+- macOS
+- Linux (32-bit & 64-bit)
+- Windows (32-bit & 64-bit)
+
+### Auto Updates
+
+Once you have the application installed it should download and alert you that a new version is available and that new version should be installed on next launch.
+
+## How to setup Reactotron in our app
+
+- [**React Native**](./docs/plugin-docs/quick-start-react-native.md#installing-reactotronapp)
+- [**React**](./docs/plugin-docs/quick-start-react-js.md#installing-reactotronapp)
+
+## How to use Reactotron's features/plugins
+
+- [**Track Global Errors**](./docs/plugin-docs/plugin-track-global-errors.md)
+- [**Async Storage**](./docs/plugin-docs/plugin-async-storage.md)
+- [**Benchmark**](./docs/plugin-docs/plugin-benchmark.md)
+- [**apisauce**](./docs/plugin-docs/plugin-apisauce.md)
+- [**Networking**](./docs/plugin-docs/plugin-networking.md)
+- [**Overlay**](./docs/plugin-docs/plugin-overlay.md)
+- [**MST**](./docs/plugin-docs/plugin-mst.md)
+- [**Redux**](./docs/plugin-docs/plugin-redux.md)
+- [**Open in Editor**](./docs/plugin-docs/plugin-open-in-editor.md)
+- [**Storybook (only for React Native)**](./docs/plugin-docs/plugin-storybook.md) \
+ `reactotron-react-native` ships with [Storybook](https://storybook.js.org/).
+This allows the switching to Storybook from the reactotron interface.
```

Comment:
>This allows the switching to Storybook from the reactotron interface.

Maybe this should say "This enables you to switch to...". Also, do we capitalize "reactotron"?